client/index.html

```diff
@@ -301,20 +301,36 @@ <h1 class="title">Settings</h1>
 								</label>
 							</div>
 							{{/if}}
+							{{#unless public}}
+							<div class="col-sm-12">
+								<h2>Push Notifications</h2>
```

Comment:
Okay, playing with this on the laptop actually shows me the usage of both, which is pretty much how I've described it.

So definitely one checkbox "Enable notifications" that enables both, based on capabilities. The only issue I see is going to be on the side of the code, as we currently disable the checkbox when `Notification` is not available. Now it's going to be a bit more intricate, but we should still be able to do all that with a single checkbox, which would be the most natural in the UI.